### ISSUE TITLE
Cannot save and close settings dialog

### ISSUE THREAD

**Issue by Seeloewen** [k-pet-group/BlueJ-Greenfoot#2403]
Since the recent version (5.4.1) I am unable to save my settings. Clicking on 'OK' in the settings dialog does not do anything. Tried on multiple environments, both Windows 11. My language is set to German. Even after reinstalling and under a fresh Windows Installation the issue persists.

**Comment by neilccbrown** [comment]
So the dialog remains showing?  We may need to see the debug log, as explained on <a href="https://bluej.org/support.html">the support page</a>.

**Comment by Seeloewen** [comment]
Yes, the dialog remains. Here is the debug log: https://pastebin.com/b8TKFyy3. Does it contain any sensitive information or can I leave it on pastebin?

**Comment by neilccbrown** [comment]
The only sensitive info in debug log only contains file paths to your projects, but you can delete it now if you wish.  The problem here seems to be that settings doesn't work while there is no project open.  This is a bug, but the work-around for now is to open a project before adjusting settings.  I will retitle the bug.
